tests/test_virtualenv_activator.py

```diff
@@ -8,9 +8,7 @@
 def test_xonsh_activator(tmp_path):
     # Create virtualenv
     venv_dir = tmp_path / "venv"
-    assert b"XonshActivator" in check_output(
-        [sys.executable, "-m", "virtualenv", str(venv_dir)]
-    )
+    check_output([sys.executable, "-m", "venv", str(venv_dir)])
```

Comment:
I am checking this

Reply:
Also there is a recent release from virtualenv which could have broken this test. https://pypi.org/project/virtualenv/20.25.2/ . Also this is not a stdlib and we have vox for supporting venvs . So I suggest moving this code to a separate project.